.eslintrc.cjs

```diff
@@ -86,6 +86,19 @@ module.exports = {
           }
         ],
 
+        // Check relative import paths are used
```

Comment:
I think we're making sure we _don't_ use relative imports?

We're trying to stick with `~` aliases instead (so files can be moved without changing their imports)

```suggestion
        // Check relative import paths use aliases
```